src/clip.sol

```diff
@@ -228,8 +228,8 @@ contract Clipper {
     function kick(
         uint256 tab,  // Debt                   [rad]
         uint256 lot,  // Collateral             [wad]
-        address usr,  // Liquidated CDP
-        address kpr   // Keeper that called dog.bark()
+        address usr,  // Address that will receive any leftover collateral
+        address kpr   // Address that will receive kick/redo incentives
```

Comment:
actually this is the parameter of `kick`, so it should just be `// Address that will receive incentives` and probably apply the same comment for `redo`

Reply:
Yeah, good point. Fixed!